src/AnimateUtil/AnimateUtil.js

```diff
@@ -48,7 +48,7 @@ class AnimateUtil {
         if (style) {
           vectorContext.setStyle(style);
         }
-        vectorContext.drawGeometry(geometry);
+        // vectorContext.drawGeometry(geometry);
```

Comment:
```suggestion
```

Reply:
The line should be commented in, commented it out by accident...